### ISSUE TITLE
net.train() again in demo_MNIST_train.py file

### ISSUE THREAD

**Issue by Dylee-y** [AlexHex7/Non-local_pytorch#48]
Hi, I started to learn pytorch a few months ago,
so I might be questioning basic stuff.

in [[Non-local_pytorch](https://github.com/AlexHex7/Non-local_pytorch)/demo_MNIST_train.py](url) file,
there was `net.train()` in the 50th line, within the training loop.
after the testing loop, `net.train()` is called once more in the 70th line.
I know that switching the network into training mode after the testing loop is important,
but why is `net.train()` in the 70th line necessary when it would be called at the beginning of the training loop anyways?

**Comment by AlexHex7** [comment]
@Dylee-y Hi.

I think that `net.train()`  in the 70th line is unnecessary. Thanks for pointing it out. And I'll delete it.

**Comment by Dylee-y** [comment]
Ohh thank you very much!
